### PR TITLE
paths & funnels: Funnel to path connection

### PR DESCRIPTION
**Pull request by EDsCODE**
## Changes

*Please describe.*  
- adds ability to click from funnel and show paths based on users at a step
*If this affects the frontend, include screenshots.*  

## How did you test this code?

*Please describe.*

### CODE REVIEW

**Comment by EDsCODE** [comment]
TODO:

- [ ] add dropoff options and make sure they work
- [x] implement the designs for denoting that you're looking at a path based on a funnel
- [ ] make the dropdown button on funnels more prominent and clickable

**Comment by EDsCODE** [comment]
I think [this update](https://github.com/PostHog/posthog/pull/6167) caused a bunch of the updated files in merging master to be reformatted

**Comment by EDsCODE** [comment]
idk why tests are failing but this PR is ready for review functionally. This only handles step based funnel paths. Still need to follow up with the dropoff options. 

@liyiy once merged we should match the styles with #6163. And we need better placement for the button

File: ee/clickhouse/views/insights.py [hunk 1]
```diff
@@ -68,10 +69,15 @@ def calculate_path(self, request: Request) -> Dict[str, Any]:
...
+        funnel_filter = None
+        funnel_filter_data = request.GET.get("funnel_filter")
+        if funnel_filter_data:
+            funnel_filter = Filter(data={"insight": INSIGHT_FUNNELS, **json.loads(funnel_filter_data)})
+
         #  backwards compatibility
         if filter.path_type:
             filter = filter.with_data({PATHS_INCLUDE_EVENT_TYPES: [filter.path_type]})
-        resp = ClickhousePaths(filter=filter, team=team).run()
+        resp = ClickhousePaths(filter=filter, team=team, funnel_filter=funnel_filter).run()
```
Comment (neilkakkar) [inline ee/clickhouse/views/insights.py:80]: Reminder to self for future PR: Need to add funnel_filter in for Path Persons too.

File: frontend/src/scenes/insights/InsightTabs/PathTab/NewPathTab.tsx [hunk 2]
```diff
@@ -43,6 +44,11 @@ export function NewPathTab(): JSX.Element {
           })
         : []
 
+    const overrideStartInput =
+        filter.funnel_paths === FunnelPathType.between || filter.funnel_paths === FunnelPathType.after
```
Comment (neilkakkar) [inline frontend/src/scenes/insights/InsightTabs/PathTab/NewPathTab.tsx:48]:
```suggestion
        [FunnelPathType.between, FunnelPathType.after].includes(filter.funnel_paths)
```

Comment (neilkakkar) [reply frontend/src/scenes/insights/InsightTabs/PathTab/NewPathTab.tsx:48]: similar below

File: frontend/src/scenes/paths/pathsLogic.ts [hunk 3]
```diff
@@ -26,14 +26,16 @@ function cleanPathParams(filters: Partial<FilterType>): Partial<FilterType> {
         end_point: filters.end_point,
         // TODO: use FF for path_type undefined
         path_type: filters.path_type ? filters.path_type || PathType.PageView : undefined,
-        include_event_types: filters.include_event_types || [PathType.PageView],
+        include_event_types: filters.include_event_types || [...(filters.funnel_filter ? [] : [PathType.PageView])],
```
Comment (neilkakkar) [inline frontend/src/scenes/paths/pathsLogic.ts:29]: Unless I'm misunderstanding this:

```suggestion
        include_event_types: filters.include_event_types || (filters.funnel_filter ? [] : [PathType.PageView]),
```

**Review by neilkakkar — COMMENTED** [review]
Looks decent!

I'll push for polishing the UI on this later - want to see how it does in prod.

**Pull request merged**